ext/vl53/vl53_platform.h

```diff
@@ -0,0 +1,219 @@
+/**
+ *
+ * Copyright (c) 2021 STMicroelectronics.
+ * All rights reserved.
+ *
+ *
+ * This software component is provided to you as part of a software package and
+ * applicable license terms are in the  Package_license file. If you received this
+ * software component outside of a package or without applicable license terms,
+ * the terms of the BSD-3-Clause license shall apply.
+ * You may obtain a copy of the BSD-3-Clause at:
+ * https://opensource.org/licenses/BSD-3-Clause
+ *
+ ******************************************************************************
+ */
+
+#ifndef VL53_PLATFORM_H_
+#define VL53_PLATFORM_H_
+#pragma once
+
```

Comment:
```suggestion
#if __has_include("modm_config_vl53.h")
#include "modm_config_vl53.h"
#endif
```
To allow user configuration without modifying this library file.

Reply:
done